server/util/getUsersFromFile.js

```diff
@@ -1,9 +1,8 @@
-const path = require('path');
-const fs = require('fs/promises');
-
 const getUsersFromFile = async file => {
-    const roleGroupPath = path.join(__dirname, file);
-    const roleGroupTxt = await fs.readFile(roleGroupPath, { encoding: 'utf8' });
+    const response = await fetch(
+        `https://raw.githubusercontent.com/w3c/aria-at-app/modify-tester-vender-access/${file}`
```

Comment:
This probably needs to change to `main` branch?

Reply:
Done!